simple_pipeline/mix.exs

```diff
@@ -28,11 +28,12 @@ defmodule Membrane.Demo.SimplePipeline.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:membrane_core, "~> 0.10"},
-      {:membrane_file_plugin, "~> 0.12"},
-      {:membrane_portaudio_plugin, "~> 0.13"},
-      {:membrane_ffmpeg_swresample_plugin, "~> 0.15"},
-      {:membrane_mp3_mad_plugin, "~> 0.13.0"},
+      {:membrane_core, "~> 0.12.0", override: true},
```

Comment:
I think we shouldn't `override` the dependencies, are there still some problems with compatibility?